publisher/monitor/price/monitor-price.py

```diff
@@ -24,14 +24,11 @@ async def main():
     channel_id = os.environ.get("SLACK_CHANNEL_ID")
 
     assets = PONTIS_ALL_ASSETS
-    os.environ["PUBLISHER_PREFIX"] = "pontis"
+    os.environ["PUBLISHER"] = "pontis"
```

Comment:
Why is this necessary?

Reply:
https://github.com/42labs/Pontis/pull/74/commits/2e6e20feb71b2ceb6b4689164847f428b500e5ff